### PR TITLE
DataVolume example

### PR DESCRIPTION
**Pull request by vatsalparekh**
**What this PR does / why we need it**:
Added example for VM with DataVolumes
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Added example for VM with DataVolumes
```

### CODE REVIEW

**Comment by mmazur** [comment]
Looks good to me, though I'd like Ondra to take a look, since he's more likely to spot mistakes.

File: tests/playbooks/kubevirt_dv_vm.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+- name: VM Creation with DataVolumes
+  hosts: localhost
+  connection: local
+  module_defaults:
+    group/k8s:
+      namespace: default
+  tasks:
+    - name: Create a VM with a DataVolume that was created
```
Comment (pkliczewski) [inline tests/playbooks/kubevirt_dv_vm.yaml]: What do we mean by  
> that was created

?

**Comment by machacekondra** [comment]
Looks ok to me, maybe fix the task name as Piotr suggests.

File: tests/playbooks/kubevirt_dv_vm.yaml (hunk 1 above)
Comment (vatsalparekh) [reply tests/playbooks/kubevirt_dv_vm.yaml]: typo from a copied playbook :laughing:

**Pull request merged**